src/main/java/nextstep/common/constant/ErrorCode.java

```diff
@@ -0,0 +1,64 @@
+package nextstep.common.constant;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ErrorCode {
+    STATION_NOT_FOUND(400, "STATION_NOT_FOUND", "역을 찾을 수 없습니다."),
```

Comment:
HTTP 응답 코드와 동일한 코드 값을 의도하셨다면, 이미 정의된 상수를 활용해보는 것도 좋을 것 같습니다.
```suggestion
    STATION_NOT_FOUND(HttpStatus.BAD_REQUEST.value(), "STATION_NOT_FOUND", "역을 찾을 수 없습니다."),
```